gms/api/src/main/pegasus/com/linkedin/ml/MLModel.pdl

```diff
@@ -0,0 +1,96 @@
+namespace com.linkedin.ml
+
+import com.linkedin.common.ChangeAuditStamps
+import com.linkedin.common.MlModelUrn
+import com.linkedin.common.VersionTag
+import com.linkedin.common.Ownership
+import com.linkedin.common.InstitutionalMemory
+import com.linkedin.common.Status
+import com.linkedin.common.Cost
+import com.linkedin.common.Deprecation
+import com.linkedin.ml.metadata.MLModelProperties
+import com.linkedin.ml.metadata.IntendedUse
+import com.linkedin.ml.metadata.MLModelFactors
+import com.linkedin.ml.metadata.Metrics
+import com.linkedin.ml.metadata.EvaluationData
+import com.linkedin.ml.metadata.TrainingData
+import com.linkedin.ml.metadata.QuantitativeAnalyses
+import com.linkedin.ml.metadata.EthicalConsiderations
+import com.linkedin.ml.metadata.CaveatsAndRecommendations
+
+
+/**
+ * MlModel spec. for a model store. A collection of MlModel metadata schema that can evolve over time.
+ */
+record MLModel includes MLModelKey, ChangeAuditStamps, VersionTag {
```

Comment:
Looks like `VersionTag` is not needed here since it's already included in `MLModelProperties` aspect? Not sure about `ChangeAuditStamps` either.
This pattern seems to be similar to dataset entity https://github.com/linkedin/datahub/blob/master/gms/api/src/main/pegasus/com/linkedin/dataset/Dataset.pdl#L11 which is some legacy code we are planning to drop soon.

Reply:
Sure, makes sense!